tests/ert_tests/dark_storage/conftest.py

```diff
@@ -57,7 +57,8 @@ def poly_example_tmp_dir(poly_example_tmp_dir_shared):
 def dark_storage_client(monkeypatch):
     with dark_storage_app_(monkeypatch) as dark_app:
         monkeypatch.setenv("ERT_STORAGE_RES_CONFIG", "poly.ert")
-        yield TestClient(dark_app)
+        with TestClient(dark_app) as client:
+            yield client
```

Comment:
I think this is the correct way of doing this. Otherwise, fastapi shutdown code won't run. We don't need that now, but probably best to do it right, in case we need it at some point.